pymllm/layers/rms_norm.py

```
@@ -26,24 +37,33 @@ def forward(
-
-        original_shape = x.shape
-        x_2d = x.reshape(-1, self.hidden_size)
-        out = flashinfer.norm.rmsnorm(x_2d, self.weight, self.eps)
-        return out.reshape(original_shape)
+        if residual is not None:
+            try:
+                flashinfer.norm.fused_add_rmsnorm(
+                    x, residual, self.weight.data, self.eps
+                )
+                return x, residual
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)